roles/create_rds_global_cluster/tasks/create.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Run 'create_rds_global_cluster' role create operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
```

Comment:
I had to add `group/cloud.amazon.aws` here too when including this role in a playbook. I think it isn't needed for the integration tests because we set in the test task, but it is needed otherwise.

Reply:
added